lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -0,0 +1,163 @@
+require 'fastlane/action'
+require_relative '../../helper/github_helper'
+
+module Fastlane
+  module Actions
+    class CreateReleaseBackmergePullRequestAction < Action
+      DEFAULT_BRANCH = 'trunk'.freeze
+
+      def self.run(params)
+        token = params[:github_token]
+        repository = params[:repository]
+        release_branch = params[:release_branch]
+        default_branch = params[:default_branch]
+        target_branches_param = params[:target_branches]
```

Comment:
> what would happen if the caller end up calling this action with target_branches: nil explicitly?

It seems Fastlane will use the default value instead of `nil`, so `[]` on this case.

Reply:
Updated on 0576800278293b6c094b5b563da5b5d96b272584